src/content-cli-config.ts

```diff
@@ -53,6 +53,21 @@ export class Config {
 
         return program;
     }
+
+    public static import(program: CommanderStatic): CommanderStatic {
+        program
+            .command("import")
+            .description("Command to import package configs")
+            .option("-p, --profile <profile>", "Profile which you want to use to list packages")
```

Comment:
to import packages?

I know you said you would tackle these maybe in a follow up PR, just commenting to not forget this.